src/ch/ehi/umleditor/application/MetaAttributePanel.java

```diff
@@ -225,13 +233,15 @@ public void setCurrentObject(ModelElement element) {
         // discard existing content
         tableModel.setRowCount(0);
 
-        for (Iterator it = element.iteratorTaggedValue(); it.hasNext(); ) {
-            TaggedValue metaAttribute = (TaggedValue)it.next();
-            String name = getMetaAttributeName(metaAttribute);
-            String value = metaAttribute.getDataValue();
+        if (element != null) {
+            for (Iterator it = element.iteratorTaggedValue(); it.hasNext(); ) {
```

Comment:
REQ falls so möglich foreach verwenden.
```suggestion
            for (TaggedValue metaAttribute : element.iteratorTaggedValue()) {
```

Reply:
Es ist leider nicht möglich mit _foreach_ durch alle Elemente eines `java.util.Iterator` zu iterieren.